src/components/sw360/AddAdditionalRoles/AddAdditionalRoles.tsx

```diff
@@ -87,9 +109,9 @@ function AddAdditionalRoles({ documentType, setDataInputList, inputList, setInpu
                                 </div>
                                 <div className='col-lg-5'>
                                     <input
-                                        name='email'
+                                        name='value'
                                         value={elem.value}
-                                        type='email'
+                                        type='text'
```

Comment:
why is the type changed to **text**?

Reply:
Thanks @hoangnt2 ! It should be `email`.